consensus/src/testing/dag.rs

```diff
@@ -139,16 +139,16 @@ impl DagFeeder {
 
     fn on_reconstructed_unit(&mut self, unit: ReconstructedUnit) {
         let h = unit.hash();
-        let parents = unit.parents();
+        let parents = unit.parents().collect::<Vec<_>>();
         let expected_hashes: HashSet<_> = self
             .units_map
             .get(&h)
             .expect("we have the unit")
             .parent_hashes()
             .into_iter()
             .collect();
-        assert_eq!(parents.item_count(), expected_hashes.len());
-        for (_, hash) in parents {
+        assert_eq!(parents.len(), expected_hashes.len());
+        for hash in parents {
```

Comment:
Hm, wouldn't now `collect`ing `parents()` into a `HashSet` actually let you just `assert_eq` on these two hashsets?

Reply:
Yes, changed

Reply:
Uhm, but you can collect them straight into a `HashSet`, no need for the `Vec` first. :P Also preferably by typing the variable, rather than the turbofish.

Reply:
Right, done